### DISCUSSION TITLE
Koinly problem

### DISCUSSION THREAD

**Discussion by algame83** [pwillworth/dfkreport#22] [General]
Something seems to have gone wrong with Koinly, I just checked in on my DFK wallet and all of the liquidity LP transactions are showing "missing purchase history". For some reason they are now disconnected from the previous transactions to create the LP. I am almost certain this was working ok a few months ago when I first imported the csv file from Lila's Ledger. 

Not sure if this is something that has changed with Koinly? Its unfortunate as I have no idea how to get Koinly to recognise these transactions as liquidity pools.

**Comment by fidomullis** [comment]
Have you tried changing the tag to 'Sent to Pool'?

**Reply by algame83** [reply]
Yes, I guess thats a work around for now, but it will not calculate any loss/gains for the in/out process as it does for supported defi like Fantom. I'm pretty sure my country legally counts liquidity transactions as taxable so its a problem.

**Comment by fidomullis** [comment]
What do you mean? It DOES calculate loss/gain if you tag withdrawals as 'Sent to Pool', and deposits as 'Received from Pool'. It's just a bit of effort to do it manually

**Reply by algame83** [reply]
From what I could see it does not process the differences when you do it manually, but maybe I missed something so I will check again. Thanks for the assistance.

**Reply by algame83** [reply]
To try and explain further there is no way (that I can see) to connect the transactions, for example: I have two earlier transactions, JEWEL and ONE both 'deposit to LP token', then further up I have the JEWEL/ONE LP deposit that was created. I can tag them all with 'sent to pool' but there is no manual way to connect them, so its still saying "missing purchase history" for the JEWEL/ONE LP and there is no cost/loss calculations.
